browser/src/core/LOUtil.js

```diff
@@ -118,11 +118,10 @@ L.LOUtil = {
 
 		map.on('themechanged', setupIcon, this);
 	},
-	setUserImage: function(img, map) {
+	setUserImage: function(img, map, myViewId) {
```

Comment:
can we rename this viewId, since as it isn't always ours?

Reply:
yes we can use just `viewId` which will make sense. Thanks i will
 change it :)